es-bs-base/scss/utilities/_text.scss

```diff
@@ -25,6 +25,36 @@
   }
 }
 
+// Size
+
+@each $size, $value in $font-sizes {
+  .font-size-#{$size} {
+    font-size: $value !important;
+  }
+}
+
+// Responsive size
+
+// TODO: Align the design system guild on font sizing, responsive font sizing,
+// and the sm/md/lg/xl/xxl font sizes. Perhaps normalize font size token names
+// to numbers (e.g. 100, 200) like Adobe.
```

Comment:
So `sm` would equal `100`, `mg` would equal `200`, ect.?

Reply:
Not sure; would have to dig into it before I had a proposal in mind. I just know that at the moment, the `font-size-{xx}` utility classes don't align with the heading sizes (nor with most font sizes I'm seeing in Figmas), we don't have the "Post 1", "Post 2", etc. marketing-size typography sizes in ESDS, and probably some other things to normalize and align.